python/ray/tune/impl/utils.py

```diff
@@ -26,3 +29,13 @@ def execute_dataset(config_dict: dict):
                     config_dict[k].categories = [
                         _item.fully_executed() for _item in _list
                     ]
+
+
+def process_scaling_config(params_space: dict):
+    """Convert ``params_space["scaling_config"]`` back to a dict so that
+    it can be used to generate search space.
+    """
+    scaling_config = params_space.get(_KEY_SCALING_CONFIG, None)
```

Comment:
Make the naming of `params_space` be consistent with `process_dataset_param(config_dict: dict)`?

Reply:
`process_dataset_param ` has a recursive impl, hence the naming.